test/cuda/tensors.jl

```diff
@@ -0,0 +1,609 @@
+using Adapt, CUDA, cuTENSOR
+using Test, TestExtras
+using TensorKit, Combinatorics
+ad = adapt(Array)
+const CUDAExt = Base.get_extension(TensorKit, :TensorKitCUDAExt)
+@assert !isnothing(CUDAExt)
+const CuTensorMap = getglobal(CUDAExt, :CuTensorMap)
+const curand = getglobal(CUDAExt, :curand)
+const curandn = getglobal(CUDAExt, :curandn)
+const curand! = getglobal(CUDAExt, :curand!)
+const curandn! = getglobal(CUDAExt, :curandn!)
```

Comment:
```suggestion
using CUDA: rand as curand, rand! as curand!, randn as curandn, randn! as curandn!
```

I think this doesn't really change anything, but I find it slightly easier to follow how these are defined, and then the tests also don't depend on the implementation details of the extension